GraphsV2/Components/utils/Legend/StandardLegend.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { styled } from '@material-ui/core/styles';
+
+const Container = styled('div')({
+    display: 'flex',
+    flexWrap: 'wrap',
+    fontSize: ({ legend: { labelFontSize } } = {}) => (
+        labelFontSize || '0.6rem'
+    ),
+    margin: '0.3rem',
+});
+
+const Item = styled('div')({
+    color: ({ color } = {}) => color,
+    flexBasis: '50%',
+    padding: '0.15rem 0',
+});
+
+export default ({ payload: legends, labelColumn, legend }) => {
+    return (
+    <Container legend={legend}>
+        {
+            legends.map(({ color, payload, value }, index) => (
+                <Item key={`item-${index}`} color={color}>
```

Comment:
it should be legend-${index} and we should have an id as well so we could add tests